src/components/AppPagination.vue

```diff
@@ -2,10 +2,8 @@
   <ul class="flex items-center -mx-2">
     <li>
       <button
-        :class="
-          isFirst &&
-          'opacity-25 p-2 bg-primary hover:bg-white hover:font-bold hover:border-link'
-        "
+        class="p-2"
+        :class="isFirst && 'opacity-25 hover:font-bold hover:border-link'"
```

Comment:
This commit also makes painfully clear that I've not worked out how to switch between different "themes", so to speak. So what made sense in our salmon-ish theme actually doesn't match designs – or, at least, the mapping isn't clear in my mind.

Reply:
Yeah we should chat about this as I'm also struggling to do the mapping sometimes and end up just guessing.

Reply:
Totes 👊